roles/slurm_workers/tasks/main.yml

```diff
@@ -114,6 +114,9 @@
   with_items:
     - "{{ groups['compute'] }}"
   when: '"compute" in group_names'
+  delegate_to: "{{ item }}"
+  with_items:
+    - "{{ play_hosts }}"
```

Comment:
Instead of building an implicit loop using `with_items:` can we simply do:
`delegate_to: "{{ play_hosts }}"`?

`with_items:` tends to be really slow.

Reply:
@lwilson,  `delegate_to` accepts only one host at a time. `delegate_to: "{{ play_hosts }}"` takes more than one host at a time. So I have used `with_items` . We have tested this code. I'll verify if we can avoid `with_items`. Can you please approve the code, I'll check if we can avoid `with_items` and once it is done I'll create a new PR.

Reply:
I looked up the documentation for `delegate_to`, and it appears to only accept a single string (https://docs.ansible.com/ansible/latest/user_guide/playbooks_delegation.html#delegating-facts).

I will approve the PR.